classes/parent-meta-image.php

```diff
@@ -15,11 +15,16 @@ public function get_group() {
     }
 
     public function get_categories() {
-        return [
+        $cats = [
             \Elementor\Modules\DynamicTags\Module::IMAGE_CATEGORY,
             \Elementor\Modules\DynamicTags\Module::MEDIA_CATEGORY,
-            \Jet_Engine_Dynamic_Tags_Module::IMAGE_CATEGORY,
         ];
+
+        if (class_exists('Jet_Engine_Dynamic_Tags_Module')) {
```

Comment:
@lukechinworth nice catch - the entire plugin is triggered by the `elementor/dynamic_tags/register_tags` action, which doesn't get called without Elementor, but not having Jet_Engine is still a possibility